src/components/teleporter.js

```diff
@@ -24,7 +24,6 @@ const RayCurve = function(numPoints, width) {
   });
 
   this.mesh = new THREE.Mesh(this.geometry, this.material);
-  this.mesh.drawMode = THREE.TriangleStripDrawMode;
```

Comment:
This causes the teleporter ray to render incorrectly. Since triangle strips are gone we will need to just adjust how we are generating the actual vertices. I can tackle that. 
![image](https://user-images.githubusercontent.com/130735/121617175-437a1700-ca19-11eb-8f63-23a94b39093c.png)

Reply:
Fixed this https://github.com/mozilla/hubs/pull/4321/commits/b3e197a427b94b56d9823e92f9f8a5759f9ac848

Reply:
Thanks!